<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="display">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>761</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>223</width>
      <height>621</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <widget class="QPushButton" name="add_in_list">
       <property name="text">
        <string>add in list</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="layot">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Surname</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="surname">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Patronymic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="patronymic">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Physics</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Physics">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Math</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="math">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Russian Language</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="russian">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="change_in_list">
       <property name="text">
        <string>change in list</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="view_file">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>300</y>
      <width>771</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>-10</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>40</y>
       <width>264</width>
       <height>174</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QComboBox" name="combo_change"/>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="search">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QComboBox" name="combo_delete"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="delete_2">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="sort">
        <property name="text">
         <string>Sort</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="combo_sort">
        <item>
         <property name="text">
          <string>Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Surname</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Patronymic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Physics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Math</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Russian Language</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="change">
        <property name="text">
         <string>Change</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="combo_search">
        <item>
         <property name="text">
          <string>Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Surname</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Patronymic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Physics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Math</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Russian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Go Back</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QLineEdit" name="search_"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="resave">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>390</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Resave in File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="read">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>350</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_file">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>310</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>320</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>600</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1330</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
